### ISSUE TITLE
Yamaha Crx 040

### ISSUE THREAD

**Issue by irplus-remote** [irplus-remote/irplus-codes.github.io#84]
http://hifi-rem.ipower.com/forums/search.php?search_author=underquark&sid=67ad62d99e9214e8e4e57136908ae1ff

 some info here

**Comment by rockeek** [comment]
The link is dead. I'm interested in having this info too! :)

**Comment by rockeek** [comment]
Thanks for that!
But from there, how can I find the IR codes for it please?
In the existing codes in your app (very good app by the way!), I can use the CRX-330 which has very similar buttons.

But I miss the button "Setup" and "Tuner" to setup the radio stations.
Can you help me finding that please? Thanks a lot !! :-)

**Comment by rockeek** [comment]
Actually, the realy question is: how to automatically Tune the radio stations.
The doc says:

![image](https://user-images.githubusercontent.com/16036588/124078971-d91d2b00-da48-11eb-835a-e1de40eda6c5.png)

But when I long press Tuning, it does not work even if I set "Repetition frequency" to 1 ms.

Do you know what I do wrong? Thanks ! :)

**Comment by irplus-remote** [comment]
Long press is a dodgy thing unfortunately, the repeat always repears the full code. 
So to do a proper long press it would surely need a codechange in irplus 

It could be saved in a separate button with the correct function for example.
Without having the device myself its very hard to replicate :/ 

0000 006D 0022 0002 0155 00AA 0015 0015 0015 0015 0015 0015 0015 0040 0015 0040 0015 0040 0015 0040 0015 0015 0015 0040 0015 0040 0015 0040 0015 0015 0015 0015 0015 0015 0015 0015 0015 0040 0015 0015 0015 0015 0015 0015 0015 0040 0015 0015 0015 0040 0015 0015 0015 0015 0015 0040 0015 0040 0015 0040 0015 0015 0015 0040 0015 0015 0015 0040 0015 0040 0015 05E9 0155 0055 0015 0E3C

![image](https://user-images.githubusercontent.com/14196521/124079729-cf47f780-da49-11eb-82ad-b6a2a34d7517.png)

**Comment by rockeek** [comment]
Thanks for the screenshot. This is the result we want? Or is it the result we get for now when I long press the button in your app?

**Comment by irplus-remote** [comment]
Yes this would simulate a 2-3 second button press. Currently its only a Single press. you can enable a signal graph in the irplus settings to show what is going out :)
